internal/pkg/context/context.go

```diff
@@ -0,0 +1,76 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package context
```

Comment:
suggest a different package name (like ldcontext). `context` is overloaded between the go context and our own usage of context in projects.

Reply:
ok, will rename